### DISCUSSION TITLE
Change the default bundle start level

### DISCUSSION THREAD

**Discussion by laeubi** [eclipse-equinox/equinox#49] [Ideas]
Currently equinox uses a default bundle start level of 4.

This is rather limited, given the current restriction that e.g. simpleconfigurator already occupies level 1.

Apache Karaf for example uses a level of 500 (where everything >= 100 is considered the user-levels) allowing a much more fine grained control of start levels.

### Proposal:
So I'd like to discuss if it might be possible that we for example use at least a start level of 100 as the default bundle start level.

### Possible side effects
Products currently rely on the default of 4 and assigning a level > 4 to **not** start a bundle might now see there bundle started and are required to adjust there level if the upgrade, for eclipse itself, as it makes no use of such things as far as I know, nothing will change.

**Comment by tjwatson** [comment]
As I stated in #48 I see no good reason to allow the default bundle start level to ever be set to something > than what the ultimate framework start-level is set to by default (which defaults to `6`).  At the very least the default bundle start level should be forced to be something < what the ultimate framework start-level is set to.  But I think the most simple change to consider would be to have the default framework start-level be something significantly more than the current default of `6`.  I had suggested 100, but we can probably go higher as long as we confirm it doesn't have performance impact to startup.
